entries/r/ramp.com.json

```
@@ -0,0 +1,17 @@
+{
+  "Ramp": {
+    "domain": "ramp.com",
+    "tfa": [
+      "sms",
```

According to the doc page, 2FA via email is also available.
```suggestion
      "sms",
      "email",
```

---

From using the service, email is available only when neither sms nor an authenticator are used (reflected on the docs page as well). That said, adding per your suggestion.